jsk_topic_tools/sample/sample_boolean_node.launch

```diff
@@ -0,0 +1,51 @@
+<launch>
+
+  <node name="is_speaking"
+        pkg="rostopic" type="rostopic"
+        args="pub -r 100 /robotsound/is_speaking std_msgs/Bool 'data: True'" />
+
+  <node name="is_speaking_jp"
+        pkg="rostopic" type="rostopic"
+        args="pub -r 100 /robotsound_jp/is_speaking std_msgs/Bool 'data: False'" />
+
+  <node name="or_node"
+        pkg="jsk_topic_tools" type="boolean_node.py" >
+    <rosparam>
+      operator: or
+    </rosparam>
+    <remap from="~input1" to="/robotsound/is_speaking" />
+    <remap from="~input2" to="/robotsound_jp/is_speaking" />
+    <remap from="~output" to="/is_speaking" />
+  </node>
+
+  <node name="and_node"
+        pkg="jsk_topic_tools" type="boolean_node.py" >
+    <rosparam>
+      operator: and
+    </rosparam>
+    <remap from="~input1" to="/robotsound/is_speaking" />
+    <remap from="~input2" to="/robotsound_jp/is_speaking" />
+    <remap from="~output" to="/both_are_speaking" />
+  </node>
+
+  <node name="xor_node"
```

Comment:
I changed my launch file like below,
```xml
<launch>

  <node name="is_speaking"
        pkg="rostopic" type="rostopic"
        args="pub -r 100 /robotsound/is_speaking std_msgs/Bool 'data: True'" />

  <node name="is_speaking_jp"
        pkg="rostopic" type="rostopic"
        args="pub -r 100 /robotsound_jp/is_speaking std_msgs/Bool 'data: True'" />

  <node name="is_speaking_kr"
        pkg="rostopic" type="rostopic"
        args="pub -r 100 /robotsound_kr/is_speaking std_msgs/Bool 'data: True'" />

  <node name="or_node"
        pkg="jsk_topic_tools" type="boolean_node.py" >
    <rosparam>
      operator: or
      number_of_input: 3
    </rosparam>
    <remap from="~input1" to="/robotsound/is_speaking" />
    <remap from="~input2" to="/robotsound_jp/is_speaking" />
    <remap from="~input3" to="/robotsound_jp_kr/is_speaking" />
    <remap from="~output" to="/is_speaking" />
  </node>

  <node name="and_node"
        pkg="jsk_topic_tools" type="boolean_node.py" >
    <rosparam>
      operator: and
      number_of_input: 3
    </rosparam>
    <remap from="~input1" to="/robotsound/is_speaking" />
    <remap from="~input2" to="/robotsound_jp/is_speaking" />
    <remap from="~input3" to="/robotsound_jp_kr/is_speaking" />
    <remap from="~output" to="/all_nodes_speaking" />
  </node>

  <node name="xor_node"
        pkg="jsk_topic_tools" type="boolean_node.py" >
    <rosparam>
      operator: xor
      number_of_input: 3
    </rosparam>
    <remap from="~input1" to="/robotsound/is_speaking" />
    <remap from="~input2" to="/robotsound_jp/is_speaking" />
    <remap from="~input3" to="/robotsound_jp_kr/is_speaking" />
    <remap from="~output" to="/xor_speaking" />
  </node>

  <node name="not_node"
        pkg="jsk_topic_tools" type="boolean_node.py" >
    <remap from="~input1" to="/robotsound/is_speaking" />
    <remap from="~output" to="/robot_is_not_speaking" />
    <rosparam>
      operator: not
      number_of_input: 1
    </rosparam>
  </node>

</launch>
```
, which means all `is_speaking*` node publishes `True`, `xor` node subscribes those all topics. To see
[wikipedia](https://ja.wikipedia.org/wiki/XOR%E3%82%B2%E3%83%BC%E3%83%88#3%E4%BB%A5%E4%B8%8A%E3%81%AE%E5%85%A5%E5%8A%9B%E3%81%B8%E3%81%AE%E6%8B%A1%E5%BC%B5) , I think `xor node` is expected to publish `True`, but it publishes `False`. Could you fix it? I'm sorry if I misunderstand something...